WooCommerce/Classes/Authentication/Prologue/LoginProloguePageViewController.swift

```diff
@@ -0,0 +1,112 @@
+import UIKit
+
+/// Displays the Login Prologue carousel, populated with `LoginProloguePageTypeViewController` pages.
+///
+final class LoginProloguePageViewController: UIPageViewController {
+
+    private var pages: [UIViewController] = []
+
+    private var pageControl: UIPageControl!
+
+    init() {
+        super.init(transitionStyle: .scroll, navigationOrientation: .horizontal)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        dataSource = self
+        delegate = self
+
+        LoginProloguePageType.allCases.forEach({ type in
+            pages.append(LoginProloguePageTypeViewController(pageType: type))
+        })
+
+        setViewControllers([pages[0]], direction: .forward, animated: false)
+
+        addPageControl()
+    }
+
+    // MARK: Page Control Setup
+    //
+    private func addPageControl() {
+        let newPageControl = UIPageControl()
+        newPageControl.currentPageIndicatorTintColor = .gray(.shade5)
+        newPageControl.pageIndicatorTintColor = .wooCommercePurple(.shade50)
+        newPageControl.transform = CGAffineTransform(scaleX: 0.8, y: 0.8)
```

Comment:
What do you think about extracting this magic number into a constant enum, to be easily changed and referenced?
Also, is this change due to the design?

Reply:
> Also, is this change due to the design?

Yeah, I was trying to size the page control according to the design. Is that ok? If we don't usually do that, and want to just have the default page control size, we can remove this line.